ParabolInc/parabol
# feat(jira-server): Save and allow to reuse recent search queries in the scope phase of poker meeting for JiraServer

Fixes: #5820

This PR introduces a new PostgreSQL table and new mutaton for storing integrations search queries.

Recent Jira Server search queries are now automatically saved

Also see the next issue with suggeted improvements: #6567

**How to test:**

- Create a poker Meeting with Jira Server integration connected
- On a Jira Server tab:
- Enter any search string and pick any found issue, see the search query was saved in the dropdown menu.
- Select any project in the project filter menu, enter any search string and pick any found issue, see the search query was saved in the dropdown menu.
- Clear the search query and the project filter, select both previous saved queries, see that they were executed correctly
- Remove queries, see they were removed correctly
